documentation/gitbook/how-to-run/multi-configs.md

```diff
@@ -32,46 +32,17 @@ you'll get a basic foward simulation of this example model. However, you might a
 flepimop simulate config_sample_2pop.yml config_sample_2pop_outcomes_part.yml
 ```
 
-if want to see what the combined configuration is, you can use the `patch` command:
+if want to create a combined configuration (e.g. to check what the combined configuration is), you can use the `patch` command:
```

Comment:
I think we should suggest this as a way to start - should look at the config before you run it. Maybe for later, should include that simulate should print out the patched config regardless - so it can be tracked what exactly was run.

Reply:
perhaps change this wording to:
> While `simulate` can run your patched configuration, we also suggest you check your configuration file using the `patch` command: